### ISSUE TITLE
Not working on Craft CMS 2.6

### ISSUE THREAD

**Issue by lboaretto** [boboldehampsink/auditlog#16]
I did a clean install and just added a word in the "Body" (Rich Text).

Only AuditLog plugin installed.
![auditlog](https://cloud.githubusercontent.com/assets/35883/15492712/9ae09942-2154-11e6-8772-7a33a16cf375.png)

I did not change the slug/URI.

Another question: This plugin works with Locale enabled?

**Comment by WilliamIsted** [comment]
I have the exact same behaviour, not a problem, but be nice if it could pick up the existing slug / URI. Also Matrix fields seem to trip it up a little.

**Comment by neekster** [comment]
Yes I'm seeing this too, quite a big issue for me as it creates many thousands of entries as I'm processing many thousands of entries via FeedMe every 15 minutes, so it creates huge amounts of log entries making it unusable for me.
